### DISCUSSION TITLE
Nfts Rewards

### DISCUSSION THREAD

**Discussion by aslanmurat4565** [IBAX-io/go-ibax#1895] [Ideas] [answered]
Will there be nft rewards? And Working on nft contract coding?

**Comment by yolandadadada** [comment, marked as answer]
Can you describe it in more detail?

**Reply by llllruzgarllll** [reply]
daha açık olabilirmisiniz

**Reply by aslanmurat4565** [reply]
Sistem nft kodlama olayında nerede acaba candy machine kullanıp nft üretebiliyor muyuz ? Mesela ben avax ağında bir launchpad projesine katıldım nft kontratı discourd üzerinde collabland kullanılarak nft kontratına whitelist imkanı sunulmuş .Bu uygulama coin tutmak yerine nft kontratına 2 avax atarak otomatik nft mint ediliyor.Mint edilen nft discourd da collabland a tanıtılınca whitelist e katılmış olunuyor.Böyle bir uygulama yapmayı düşünüyor musunuz ?

**Reply by aslanmurat4565** [reply]
> 
Where is the system in nft coding, can we use candy machine to produce nft? For example, I participated in a launchpad project on the avax network. By using collabland on the nft contract  in the discourd, whitelisting is provided for the nft contract. This application allows automatic nft mint by adding 2 avax to the nft contract instead of holding coins. Are you considering making an application?
